.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,4 +1,8 @@
-# Issue Template
+  ---
+name: "Bug Report"
+labels: bug
+assignee: kaikai1024
```

Comment:
Remove your id from the Template?

Reply:
assignee: creates an issue and assigns it to me automatically. It's the default config and can be editted.

* I want to know it ASAP and then re-assignee someone
* I think It's better to respond the issue ASAP.